### ISSUE TITLE
FFXIV runs poorly in dx11 mode compared to dx9

### ISSUE THREAD

**Issue by ashkitten** [doitsujin/dxvk#1785]
in dx11 mode the graphics settings seem to barely affect framerate at all (only about a 10 fps difference between minimum and maximum), and I get about 70fps or so in the lavender beds, dropping to sub-50 in crowded places like limsa lominsa aetheryte. in dx9 mode, it maintains a fairly stable 110fps in the lavender beds on maximum settings, getting up to 210fps when turned down to minimum.

### Software information
maximum settings, wine fsync enabled

### System information
- GPU: AMD RX 5700
- Driver: AMDGPU (Linux 5.8.10-zen) / Mesa 20.1.7 / RADV (ACO)
- Wine version: lutris-5.7.10
- DXVK version: v1.7.1-2-g743f309

### Log files
https://gist.github.com/ashkitten/ddbf3097df96f2391c0d1fac63404807

**Comment by doitsujin** [comment]
D3D11 is consistently faster than D3D9 mode on my end, not sure what your problem is.

**Comment by ashkitten** [comment]
minimum graphics settings should be comparable though, right? i only get around 80-90fps in the lavender beds on dx11 while dx9 gets over 200. i've seen the `fsync: up and running.` message, so unless there's another way to check if it is working i don't know...

**Comment by doitsujin** [comment]
And are you CPU-bound or GPU-bound?

I have absolutely no info to work with right now, other than "works fine for me".

**Comment by doitsujin** [comment]
You could also try running the old [Heavensward](https://www.guru3d.com/files-details/final-fantasy-xiv-heavensward-benchmark-download.html) benchmark with both D3D9 (max settings) and D3D11 ("max settings with d3d9") and compare; if there's a significant discrepancy then there might be a driver issue, if not, then there simply is no problem.

**Comment by ashkitten** [comment]
seems like they get very similar scores on that benchmark. i realized that while i have fsync working in the lutris build of wine (which i had to use to get dx9 working) it doesn't log anything about fsync in the wine-staging system build, so it's possible that might be the issue for me (along with dx9 not supporting the same settings). sorry for the noise, then.

**Comment by RodrigoLeiteF** [comment]
I'm experiencing the same issues on similar hardware (RX 5700 XT + Ryzen 5 3600). Esync is enable and confirmed running on lutris logs.

Wine build: lutris-5.7-10-x86_64

DirectX 11 Maximum settings: ~100FPS
DirectX 11 Minimum settings (Standard Laptop): ~105FPS

DirectX 9 Maximum settings: ~145FPS
DirectX 9 Minimum settings (Standard Laptop): ~200FPS

I tried setting the maximum tesselation level to 8 (as AMD cards are pretty bad with tesselation IIRC) but it didn't seem to help. I also enabled the DXVK HUD to take a look at what's going on, and it seems like with DX11 Minimum Settings, my GPU usage never goes past ~65%. Does this help in any way?

https://cdn.discordapp.com/attachments/201788332743720960/768233827347726396/unknown.png
